configs/eslint-common-cypress.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": [
```

Comment:
I'm wondering... Should we not be extending `eslint-ts-overrides.json` here too?

That config extends `@typescript-eslint/recommended`, which I am sure will provide some useful rules

Reply:
I don't think it should. This file contains only Cypress specific configuration. Once this file is extended by `.eslintrc.json` in `cypress` directory the rest of the configuration is inherited from the package's top level `.eslintrc.json`, which in case of ui-common/stage/composer extends `eslint-common.json`, which in turn extends `eslint-ts-overrides.json`. So configuration inheritance already takes care of this.

Reply:
Today I learned that ESLint merges subdirectory configs with the configs from parent directories :smile: Thanks!

Resources for whoever will be looking at this thread in the future and did not know it: https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy